src/containers/Network/Validators.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import { useTranslation } from 'react-i18next'
+import NetworkTabs from './NetworkTabs'
+import Streams from '../shared/components/Streams'
+import ValidatorsTable from './ValidatorsTable'
+import Log from '../shared/log'
+import { localizeNumber, FETCH_INTERVAL_MILLIS } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+import Hexagons from './Hexagons'
+
+export const Validators = () => {
+  const language = useLanguage()
+  const { t } = useTranslation()
+  const [vList, setVList] = useState<any>([{}])
+  const [validations, setValidations] = useState([])
+  const [metrics, setMetrics] = useState({})
+  const [unlCount, setUnlCount] = useState(0)
+
+  useEffect(() => {
+    fetchData()
+    const interval = setInterval(fetchData, FETCH_INTERVAL_MILLIS)
+    return () => {
+      clearInterval(interval)
+    }
+  }, [])
+
+  const fetchData = () => {
+    const url = '/api/v1/validators?verbose=true'
```

Comment:
nit: this`fetchData` function is used also in `UpgradeStatus.tsx`, can it be extracted and passed in as a pop in `index.js`?

Reply:
That is true.  We eliminated quite a bit of duplication. @pdp2121 we should target that when we add a chart for nodes.